lifecycle-operator/main.go

```diff
@@ -117,7 +118,14 @@ func main() {
 	}
 	provider := metric.NewMeterProvider(metric.WithReader(exporter))
 	meter := provider.Meter("keptn/task")
-	keptnMeters := telemetry.SetUpKeptnTaskMeters(meter)
+
+	keptnLifecycleActive, err := meter.Int64Counter("keptn_lifecycle_active")
+
+	if err != nil {
+		panic(err)
```

Comment:
please do not panic in the main function, use something like this instead

	setupLog.Error(err, "unable to ...")
		os.Exit(1)

Reply:
Alright

Reply:
Resolved